Macro compile error

I encounter below errors while I run `swift test`, is there any missing configurations?

```
~/SwiftAST/Sources/SwiftAST/Macros.swift:9:7: warning: external macro implementation type 'SwiftASTMacros.SwiftASTExpressionMacro' could not be found for macro 'ast_expandExpression'; plugin for module 'SwiftASTMacros' not found
 7 | /// returned, and an error being raised in the editor.
 8 | @freestanding(expression)
 9 | macro ast_expandExpression<T>(_: T) -> Expression =
   |       `- warning: external macro implementation type 'SwiftASTMacros.SwiftASTExpressionMacro' could not be found for macro 'ast_expandExpression'; plugin for module 'SwiftASTMacros' not found
10 |     #externalMacro(module: "SwiftASTMacros", type: "SwiftASTExpressionMacro")
11 |
```

--- a/Package.resolved
+++ b/Package.resolved
@@ -1,5 +1,5 @@
 {
-  "originHash" : "9edc6074f625350bf031cfa090db3d26d2a671fa3eef593be6e724b55a8958e6",
+  "originHash" : "c0090b43291a33db394388452d1a81d7edd35960c5e4ee3e9c5942aca54e18c6",
   "pins" : [
     {
       "identity" : "minidigraph",

--- a/Package.swift
+++ b/Package.swift
@@ -4,6 +4,7 @@ import CompilerPluginSupport
 
 let package = Package(
     name: "SwiftAST",
+    platforms: [.macOS(.v10_15), .iOS(.v13), .tvOS(.v13), .watchOS(.v6), .macCatalyst(.v13)],
     products: [
         .library(
             name: "SwiftAST",
@@ -24,11 +25,7 @@ let package = Package(
         .macro(
             name: "SwiftASTMacros",
             dependencies: [
-                .product(name: "SwiftSyntax", package: "swift-syntax"),
-                .product(name: "SwiftSyntaxBuilder", package: "swift-syntax"),
-                .product(name: "SwiftParser", package: "swift-syntax"),
                 .product(name: "SwiftSyntaxMacros", package: "swift-syntax"),
-                .product(name: "SwiftSyntaxMacrosTestSupport", package: "swift-syntax"),
                 .product(name: "SwiftCompilerPlugin", package: "swift-syntax"),
             ]
         ),
@@ -62,6 +59,7 @@ let package = Package(
             name: "SwiftASTMacrosTests",
             dependencies: [
                 "SwiftASTMacros",
+                .product(name: "SwiftSyntaxMacrosTestSupport", package: "swift-syntax"),
             ]
         )
     ]

--- a/Tests/SwiftASTTests/PatternTests.swift
+++ b/Tests/SwiftASTTests/PatternTests.swift
@@ -90,7 +90,7 @@ class PatternTests: XCTestCase {
 // MARK: - Test internals
 
 private func assertSerializeRoundabout(
-    _ sut: Pattern,
+    _ sut: SwiftAST.Pattern,
     file: StaticString = #file,
     line: UInt = #line
 ) {

--- a/Tests/SwiftASTTests/Statement/ConditionalClauseTests.swift
+++ b/Tests/SwiftASTTests/Statement/ConditionalClauseTests.swift
@@ -42,6 +42,6 @@ private func makeSut(_ clauses: [ConditionalClauseElement]) -> ConditionalClause
     .init(clauses: clauses)
 }
 
-private func makeElement(pattern: Pattern? = nil, expression: SwiftAST.Expression) -> ConditionalClauseElement {
+private func makeElement(pattern: SwiftAST.Pattern? = nil, expression: SwiftAST.Expression) -> ConditionalClauseElement {
     .init(pattern: pattern, expression: expression)
 }

